cmd/install/assets/hypershift_operator.go

```diff
@@ -835,6 +835,16 @@ func (o HyperShiftOperatorClusterRole) Build() *rbacv1.ClusterRole {
 				Resources: []string{"virtualmachineinstances", "virtualmachines"},
 				Verbs:     []string{"*"},
 			},
+			{
```

Comment:
Can we add a comment on why this is needed? e.g.
"Gran permissions needed by CPO to mirror resources from underlying cluster for kubevirt CSI driver" if that's why?

Reply:
yep, i added a comment